.github/README.md

```diff
@@ -0,0 +1,65 @@
+# :fire: react-dev-cli :fire:
+
+A command line tool that helps automate repeatable actions during react application development, like creating a new component or reducer structure. :construction_worker:
+
+## Install :hammer:
+>```npm install -g react-dev-cli```
+
+or
+>```yarn global add react-dev-cli```
+
+### Config :wrench:
+To define config create a `.rdc` file in the root directory of your application.
+
+| Property | Sample value | Description                            |
+|----------|--------------|----------------------------------------|
+| root     | src          | Path to the source of your application |
+
+## Usage :speedboat:
+:exclamation::exclamation::exclamation:
+You should run a CLI tool being always in the root directory of your application.
+
+>```rdc --help```
```

Comment:
should we really show outputs of `--help` params in docs? it can quickly get outdated, and this is quite a popular param, maybe we should show only behavior and usage of main commands, like creating component

Reply:
I think the output of the main `--help` could be useful to see what options you will have available after installation of this package. Also, keep documentation up to date should be in the scope of each task.